Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -561,12 +561,22 @@ For additional best practices and detailed information please refer to the follo
 
 ### Testing Login Activity and Device Blocking (MSTG-AUTH-11)
 
-For applications which require L2 protection, the MASVS states that: "The app informs the user of all login activities with their account. Users are able view a list of devices used to access the account, and to block specific devices.". This can be broken down into various scenarios:
+For applications which require L2 protection, the MASVS states that: "The app informs the user of all login activities with their account. Users are able view a list of devices used to access the account, and to block or allow specific devices.". 
```

Comment:
I would not include here what the MASVS says, if we start putting this in several test cases we end up having to take care of updating these snippets as well :/

Reply:
Makes sense. i will remove this and remplace it with something similar but not from the MASVS.